.github/workflows/build.yml

```diff
@@ -20,11 +20,11 @@ jobs:
         with:
           node-version: ${{ steps.nvm.outputs.NODE_VERSION }}
       - name: Install dependencies
-        run: npm ci
+        run: yarn
       - name: Build
-        run: npm run build
+        run: yarn run build
```

Comment:
Nit; I believe you can omit `run` from `yarn` commands; ie, `yarn dev` rather than `yarn run dev`. `yarn run` still works, though!